### ISSUE TITLE
Dropdown: Insufficient color contrast for sub-menu items

### ISSUE THREAD

**Issue by sirrah-tam** [ithaka/pharos#492]
**Expected behavior**
The minimum color contrast ratio for regular text is 4.5:1.

**Actual behavior**
The color token used for the sub-menu item text within the navigation panel is marble-gray-50 (#81807e) which against the white (#ffffff) background results in a 3.9:1 color contrast ratio.

Additionally, when the user hovers over the item, the background changes to marble-gray-94 (#F2F1ED) which results in a 3.5:1 color contrast ratio.

**Steps to reproduce the issue**

1. Navigate to main navigation and locate the Browse menu option
2. Activate the sub-navigation menu to appear
3. Observe "Journals and books" content
4. Using contrast tool, review text (foreground) color against the background color.

**Screenshots or code**
Example from jstor.org
![image](https://user-images.githubusercontent.com/6874453/217592649-139beb4a-c162-4e8c-9ea8-745387dc86c9.png)

**Your environment**

- OS: macOS, Windows
- Browser: Chrome, Firefox, Safari
- Version: 97.0, 105.0.2, 16.0

**Additional information**
Recommendation: Update the color for this text to marble-gray-40 (#676765) which has a contrast ratio of 5.7:1 against white background and 5:1 against hover background.

**Comment by daneah** [comment]
@sirrah-tam one wrinkle in this: The base text color can be changed here in Pharos, but the color you describe on hover is defined in the application. The default color on hover is `interactive-primary` (which you can see in action in Storybook).

We can change it to `marble-gray-40` for the base, and we can also change it in the application for the hover color to be `marble-gray-40`, or we can remove the override there and let it be the default `interactive-primary`. Curious about your thoughts and thoughts from @david-corneail.

**Comment by david-corneail** [comment]
@daneah @sirrah-tam sorry the delay, I can confirm that the product designers agreed that we should remove the override.. so the description should have interactive-primary on hover. 
![Screen Shot 2023-03-07 at 2 41 20 PM](https://user-images.githubusercontent.com/104939119/223538620-7fe96a2b-a522-4b33-9d8c-327ef362a149.png)

**Comment by daneah** [comment]
Closing this issue as we believe it is impacted best at the consumer level.
